app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -221,6 +221,22 @@ public long getDuration() {
         }
     }
 
+    public long getBufferedPosition() {
+        if (!isInitialized())
+            return -1;
+
+        // to do: add libVLC support
```

Comment:
todo comments should be ` // TODO ` or `// FIXME`.

Reply:
Good to know for the future. I turned this comment from a TODO into a "why this is exoplayer only" instead